### PR TITLE
JSUI-3101 Modified the breadcrumb aria-label values to match the facet values labels 

### PR DESCRIPTION
**Pull request by jlblais**
https://coveord.atlassian.net/browse/JSUI-3101

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

File: unitTests/ui/BreadcrumbValuesListTest.ts [hunk 1]
```diff
@@ -131,6 +131,14 @@ export function BreadcrumbValueListTest() {
         expect($$(captions[1]).text()).toContain('foo1');
         expect($$(captions[2]).text()).toContain('foo2');
       });
+
+      it('should display a label with the right attributes for each excluded values', () => {
+        const built = new BreadcrumbValueList(facet, facetValues, BreadcrumbValueElement).build();
+        const values = $$(built).findAll('.coveo-facet-breadcrumb-value')[2];
+        expect($$(values).getAttribute('aria-label')).toBe('Remove exclusion filter on foo2');
+        expect($$(values).getAttribute('role')).toBe('button');
+        expect($$(values).getAttribute('tabindex')).toBe('0');
```
Comment (jlblais) [inline unitTests/ui/BreadcrumbValuesListTest.ts:140]: Not sure about that test. I didn't see any test for the aria-label when a facet is excluded. Thought I might add one.

File: strings/strings.json [hunk 2]
```diff
@@ -754,8 +754,8 @@
     "zh-tw": "切換到 {0}"
   },
   "Unexclude": {
-    "en": "Unexclude {0}",
-    "fr": "Réinclure {0}",
+    "en": "Remove exclusion filter on {0}",
+    "fr": "Enlever le filtre d'exclusion sur {0}",
     "cs": "Znovu zahrnout {0}",
```
Comment (ThibodeauJF) [inline strings/strings.json:759]: I think it's okay to keep the other languages here, it's similar enough, and better than having a non-translated one

File: unitTests/ui/BreadcrumbValuesListTest.ts (hunk 1 above)
Comment (ThibodeauJF) [reply unitTests/ui/BreadcrumbValuesListTest.ts:140]: Thats nice, you can rename the test like the others:
`has the right accessibility attributes`

**Comment by ThibodeauJF** [comment]
Looks good!

One small improvement, looks like some tests were forgotten previously for those breadcrumbs:
- DynamicFacetBreadcrumbs
- DynamicHierarchicalFacetBreadcrumb

Those are truly button elements, so the role & the tabindex property is not necessary.

**Review by fbeaudoincoveo — APPROVED** [review]

**Review by ThibodeauJF — APPROVED** [review]
Good!

**Pull request merged**